smtplib.SMTPNotSupportedError: STARTTLS extension not supported by server.

测试工作流正常，但是主工作流出现问题。

```python
Rendering Email: 100%|██████████| 76/76 [1:55:18<00:00, 91.03s/it]
20***-01-07 00:14:29.966 | INFO     | __main__:<module>:177 - Sending email...
Traceback (most recent call last):
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/main.py", line 178, in <module>
    send_email(args.sender, args.receiver, args.sender_password, args.smtp_server, args.smtp_port, html)
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/construct_email.py", line 154, in send_email
    server.starttls()
  File "/home/runner/.local/share/uv/python/cpython-3.11.10-linux-x86_64-gnu/lib/python3.11/smtplib.py", line 771, in starttls
    raise SMTPNotSupportedError(
smtplib.SMTPNotSupportedError: STARTTLS extension not supported by server.
Error: Process completed with exit code 1.
```

--- a/construct_email.py
+++ b/construct_email.py
@@ -6,6 +6,7 @@ from email.mime.text import MIMEText
 from email.utils import parseaddr, formataddr
 import smtplib
 import datetime
+from loguru import logger
 
 framework = """
 <!DOCTYPE HTML>
@@ -152,7 +153,9 @@ def send_email(sender:str, receiver:str, password:str,smtp_server:str,smtp_port:
     try:
         server = smtplib.SMTP(smtp_server, smtp_port)
         server.starttls()
-    except smtplib.SMTPServerDisconnected:
+    except Exception as e:
+        logger.warning(f"Failed to use TLS. {e}")
+        logger.warning(f"Try to use SSL.")
         server = smtplib.SMTP_SSL(smtp_server, smtp_port)
 
     server.login(sender, password)

